system/modules/isotope/library/Isotope/Model/Document/Standard.php

```diff
@@ -167,19 +167,16 @@ protected function generatePDF(IsotopeProductCollection $objCollection, array $a
      */
     protected function generateTemplate(IsotopeProductCollection $objCollection, array $arrTokens)
     {
-        $objPage = PageModel::findWithDetails($objCollection->page_id);
-
         /** @var Template|\stdClass $objTemplate */
         $objTemplate = new Template($this->documentTpl);
         $objTemplate->setData($this->arrData);
 
         $objTemplate->title         = StringUtil::parseSimpleTokens($this->documentTitle, $arrTokens);
         $objTemplate->collection    = $objCollection;
         $objTemplate->config        = $objCollection->getConfig();
-        $objTemplate->page          = $objPage;
-        $objTemplate->dateFormat    = $objPage->dateFormat ?: $GLOBALS['TL_CONFIG']['dateFormat'];
-        $objTemplate->timeFormat    = $objPage->timeFormat ?: $GLOBALS['TL_CONFIG']['timeFormat'];
-        $objTemplate->datimFormat   = $objPage->datimFormat ?: $GLOBALS['TL_CONFIG']['datimFormat'];
```

Comment:
Note: `tl_iso_product_collection` does not have a field called `page_id`, so this never actually worked.